src/unitxt/api.py

```diff
@@ -198,6 +198,13 @@ def infer(
         )
     predictions = post_process(raw_predictions, dataset)
     if return_data:
+        if return_meta_data:
+            infer_output_list = [
+                infer_output.__dict__ for infer_output in infer_outputs
+            ]
+            for infer_output in infer_output_list:
+                del infer_output["prediction"]
+            dataset = dataset.add_column("infer_meta_data", infer_output_list)
```

Comment:
@elronbandel - this part was removed in a previous commit, which breaks the llmasjudge. please allow to re-add it here.